src/data/Data.js

```diff
@@ -577,9 +577,10 @@ export const WorkspaceData = _.flow(
     const region = isAzureWorkspace ? storageDetails.azureContainerRegion : storageDetails.googleBucketLocation;
 
     const wdsDataTableProvider = useMemo(() => {
-      const proxyUrl = !!wdsProxyUrl && wdsProxyUrl.state;
+      const app = wdsApp?.state;
```

Comment:
should this be called `appState`?

Reply:
Confusingly, no...the `wdsApp` is a `useState` that has a 'status' indicating whether it's loaded or not, and a `state` which is the stateful contents and happens to be an app.